src/main/java/org/fundacionjala/sfdc/pageobjects/SalesForceAccounts.java

```diff
@@ -0,0 +1,214 @@
+package org.fundacionjala.sfdc.pageobjects;
+
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.fundacionjala.sfdc.commons.DriverManager;
+
+/**
+ * SalesForceAccounts.java
+ * Class that represents Accounts page in SalesForce.
+ */
+public class SalesForceAccounts {
+    private WebDriver webDriver;
+    private By newAccountButton;
+    private By accountNameTextField;
+    private By saveNewAccountButton;
+    private By newAccountLabel;
+    private By accountsViewList;
+    private By goButton;
+    private By lastAccountLink;
+    private By deleteButton;
+    private By editButton;
+    private By homeContact;
+    private WebDriverWait wait;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "wait" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)